examples/manipulating_visualizing/plot_neurovault_meta_analysis.py

```diff
@@ -0,0 +1,104 @@
+"""
+NeuroVault meta-analysis of stop-go paradigm studies.
+=====================================================================
+
+This example shows how to download statistical maps from
+NeuroVault
+
+See :func:`nilearn.datasets.fetch_neurovault` documentation for more details.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn
+from nilearn import datasets
+
+# Define pre-download filters
+cfilts = []  # [lambda col: col['DOI'] is not None]
+imfilts = [lambda im: "stop signal" in (im.get('cognitive_paradigm_cogatlas') or "").lower(),
+           lambda im: np.all([s in (im.get("contrast_definition") or "")
+                              for s in ["succ", "stop", "go"]])]
+
+
+# Download up to 100 matches
+ss_all = datasets.fetch_neurovault(collection_filters=cfilts,
+                                   image_filters=imfilts)
+images, collections = ss_all['images'], ss_all['collections']
+print("Paradigms we've downloaded:")
+for im in images:
+    print("\t%04d: %s" % (im['id'],
+                          im['cognitive_paradigm_cogatlas']))
+
+# Show all contrast definitions
+print("Contrast definitions for downloaded images:")
+for cd in np.unique([im['contrast_definition'] for im in images]):
+    print("\t%s" % cd)
+
+# Visualize the content
+from nilearn.plotting import plot_glass_brain
+for im in images:
+    plot_glass_brain(im['local_path'], title='%04d' % im['id'])
+plt.show()
```

Comment:
nitpick, but I think that I would rather avoid having calls to 'show' in the middle.